scripts/sample-data.js

```diff
@@ -30,10 +30,18 @@ const NAMES = [
   '계룡/鷄龍',
   'Urist'
 ]
-const CATEGORIES = ['Housing', 'Collectibles', 'For Sale', 'Services']
+const CATEGORIES = {
+  forSale: ['artsCrafts', 'farmGarden', 'heavyEquipment', 'tickets'],
```

Comment:
Double checked now and I see translated listings stored in elasticsearch index:
<img width="789" alt="screenshot 2018-09-18 19 02 28" src="https://user-images.githubusercontent.com/579910/45703952-089f6900-bb76-11e8-95c0-133339a34615.png">

<img width="951" alt="screenshot 2018-09-18 19 04 25" src="https://user-images.githubusercontent.com/579910/45703871-caa24500-bb75-11e8-8dc9-56495f69e947.png">

Has anybody already thought through how we should handle localisation? @DanielVF @franckc

Reply:
@sparrowDom Could these be old documents that were index prior to Daniel's change ?
Regarding localization I don't think we should localize cat/subcat when indexing. These are just to be used as filters for searches.

Reply:
Yeah @franckc you are right. It makes no sense to localize them, since it is only going to work for english language. I guess we can solve that once we sort out localisation within our schemas. 

@DanielVF maybe just change them to full translation strings like `schema.forSale.antiques` instead of `antiques` https://github.com/OriginProtocol/origin-dapp/blob/master/public/schemas/for-sale.json#L27-L70

Reply:
@sparrowDom @franckc - I think in our meeting about schemas a few weeks ago, we decided not to try to localize any strings in `origin-js`, and that we would only localize strings in the schemas in `origin-dapp`. I'm not sure if the localization setup that we currently have would support translation of strings in `origin-js` but I can look into it if we're now considering it.

Reply:
@jordajm  What I meant by localisation in schemas is not that we would translate part of schemas using localisation module, but that our schemas support different languages. (sorry for lack of clarity)

Currently there is no way of storing the same listing (with the same id) in different languages. We have discussed at some point that language specific fields (like title, description ....) will be in an array of objects where each object would have a corresponding language. 

And once we support that, we will also need to change indexing/search engine quite a lot, and we can figure out then how to index data. 

But for now, that we just keep untranslated keys (like `schema.forSale.antiques`) that define schema `cateogry` and `subCategory`